api/openapi/v1/support-notifications.yaml

```diff
@@ -74,23 +82,23 @@ paths:
                 $ref: '#/components/schemas/Error'
   /v1/notification/age/{age}:
     delete:
-      description: Delete the proccessed Notifications if the current timestamp minus
+      description: Delete the proccessed notifications if the current timestamp minus
```

Comment:
I think this is a bad description that goes into way too much implementation detail. Please remove a lot of that detail and focus on what it does and not why it does it.